### ISSUE TITLE
Yone E white screen

### ISSUE THREAD

**Issue by koobzaar** [koobzaar/lol-skins#21]
We are aware that some of Yone's skins cause the screen to blink white. We have already deployed fixes for the base skins, but we still need people to work on the chromas. Any future updates regarding Yone's chromas will be delivered here.

**Comment by Disgrace0310** [comment]
Hey, how I fixed the issue on skins I posted was:
Changing the path to skin0 here
mResourceResolver: link = "Characters/Yone/Skins/Skin58/Resources"
Along with the usual, you just have to edit both lines that have ResourceResolver in them, no clue why lol

**Comment by Disgrace0310** [comment]
> I've reverted everything to your skins xD
> 
> In these cases it's better to review manually. The only problem tho is the chromas

I wouldn't name the chromas, any skin spotlight video with chromas or ingame chroma list is in order with the game files, so chroma 1 will usually be the first one on a skin spotlight video or the first chroma in champ select, I think everyone would figure it out if they were just numbered, a problem can appear with for example mythic chromas, for battle academia ezreal it was skin21 and the base was skin11? maybe it goes in incrimentals of 10

**Comment by Disgrace0310** [comment]
> I wouldn't name the chromas, any skin spotlight video with chromas or ingame chroma list is in order with the game files, so chroma 1 will usually be the first one on a skin spotlight video or the first chroma in champ select, I think everyone would figure it out if they were just numbered, a problem can appear with for example mythic chromas, for battle academia ezreal it was skin21 and the base was skin11? maybe it goes in incrimentals of 10

And in case of yone I guess it'd work fine if you'd do what I said with chromas? I made a couple and hadn't encountered any problems

**Comment by koobzaar** [comment]
Oh don't worry about the chromas ID. We already figured it out. The only problem is the color.

But we wrote a tool to interpolate the colors and match with a dictionary that we have. Still a slightly mismatch but it works well

**Comment by Disgrace0310** [comment]
> Oh don't worry about the chromas ID. We already figured it out. The only problem is the color.
> 
> But we wrote a tool to interpolate the colors and match with a dictionary that we have. Still a slightly mismatch but it works well.
> 
> When I said that the problem was the chromas I meant that we fixed the Yone normal skins flashbang issue, but this issue stands still in the chromas

Let me check it out, I'll get back to you in a sec

**Comment by Disgrace0310** [comment]
Yup, as I said, for any yone skin/chroma I just edit both lines in the Skins file
These are the lines that need to be changed for yone

    "Characters/Yone/Skins/Skin0" = SkinCharacterDataProperties {
    "Characters/Yone/Skins/Skin0/Resources" = ResourceResolver {
        mResourceResolver: link = "Characters/Yone/Skins/Skin0/Resources"

It works for chromas, just tested it with Noonday chroma with High Noon Yone
ofc use the base animations but for the chroma skin file edit these 3 :)

**Comment by koobzaar** [comment]
Hello @Disgrace0310. Lets separate into topics:
> It works for chromas, just tested it with Noonday chroma with High Noon Yone
ofc use the base animations but for the chroma skin file edit these 3, usually only 2 lines are edited and that's why it didn't work there :)

Thanks for the head's up! We will implement this into the next batch of skins. Since this problem only affects Yone chromas right now, we are not actually too much worried about it.

> edit edit: Will the program be publically accessible once it's finished? I'd much rather be able to extract all the skins myself than wait for an update to the github :P
Yes! Me and the people that are working with me believes that every knowledge shoud be free. We didn't make the repository public yet because one member of the team thinks that we should be make available when we fix some stuff. It's kinda messy right now. We keep changing the code to fix some particular issues from some specifics champions instead of making a universal solution. Maybe tomorrow (?) Idk
I keep pressing him to release it almost everyday, but I need to respect his decision.
![image](https://github.com/user-attachments/assets/c7a09aba-64f0-41fe-b7f3-a64630731167)

**Comment by Disgrace0310** [comment]
Gotcha, I also think that you shouldn't try to rush it haha, it's best to make one polished project rather than a mess that will need to get tweaked a bunch to function properly. When everyone will be satisfied with releasing it that's when it should be released, take care for now!

**Comment by koobzaar** [comment]
Hey @Disgrace0310 btw you are not on the list of the collaborators on the bottom right of the screen because you never created a pull request - you've always submitted an issue.

I feel bad for it because you helped a lot. So, could you please create a pull request with a random .txt file so I can accept it and add you to the contributors?

Clone the repository and add it anywhere.
